source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,606 @@
+   from the :ref:`Atlas Search <fts-top-ref>` documentation.
+
+The examples in this guide use the following documents in a collection called
+``guitars``:
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "description": "Classic guitars known for their versatility.", "establishedYear": 1946, "in-stock": true, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "description": "Classic guitars known for their rich, full tones.", "establishedYear": 1902, "in-stock": true, "rating": 8 }
+   { "_id": 3, "make": "PRS", "description": "High-end guitars known for their quality.", "establishedYear": 1985, "in-stock": true, "rating": 9 }
+   { "_id": 4, "make": "Kiesel", "description": "Quality guitars made only for custom orders.", "establishedYear": 2015, "in-stock": false }
+   { "_id": 5, "make": "Ibanez", "description": "Well-crafted guitars used by many professional guitarists.", "establishedYear": 1957, "in-stock": true, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "description": "Modern guitars known for their headless models.", "establishedYear": 1982, "in-stock": false }
+
+The following ``Guitar`` class models the documents in this collection.
+
+.. code-block:: csharp
+
+   public class Guitar
+   {
+       public int Id { get; set; }
+       public string Make { get; set; }
+       public List<string> Models { get; set; }
+       public int EstablishedYear { get; set; }
+       [BsonElement("in-stock")]
+       public bool InStock { get; set; }
+       public int? Rating { get; set; }
+   }
+
+.. note::
+
+   The documents in the ``guitars`` collection use the camel-case naming
+   convention. The examples in this guide use a ``ConventionPack``
+   to deserialize the fields in the collection into Pascal case and map them to
+   the properties in the ``Guitar`` class.
+
+   To learn more about custom serialization, see :ref:`csharp-custom-serialization`.
+
+Create an Atlas Search Index
+----------------------------
+
+Before you can perform a search on an Atlas collection, you must first create an **Atlas
+Search index** on the collection. An Atlas Search index is a data structure that
+categorizes data in a searchable format. 
+
+To learn how to create an Atlas Search Index see the
+:atlas:`Create an Atlas Search Index </atlas-search/create-index>` Atlas guide.
+
+Atlas Search Operators
+----------------------
+
+The ``Search`` class contains methods you can use to perform ``$search``
+operations. For a full list of available ``$search`` operations, see the :atlas:`Operators and Collectors
+</atlas-search/operators-and-collectors>` Atlas guide.
+
+.. note:: 
+
+   The ``Search`` class does not currently support the ``$embeddedDocument`` operator.
+
+Autocomplete
+~~~~~~~~~~~~
+
+Use the ``Autocomplete`` method to search for a word or phrase that contains a
+sequence of characters from an incomplete input string. The search returns
+results based on the tokenization strategy specified when creating the index.
```

Comment:
I removed that as I think it makes the description more confusing. Since the rest of the examples are just giving a basic description and referring to the Atlas docs for more information, I think it makes sense to do that here as well.